### PR TITLE
Use english name of subsitution variables for thirdparties

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
Use english name of subsitution variables for thirdparties

### CODE REVIEW

File: htdocs/core/lib/functions.lib.php [hunk 1]
```diff
@@ -6059,41 +6070,67 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
...
+				$substitutionarray['__THIRDPARTY_ZIP__'] = (is_object($object)?$object->zip:'');
+				$substitutionarray['__THIRDPARTY_TOWN__'] = (is_object($object)?$object->town:'');
+				$substitutionarray['__THIRDPARTY_IDPROF1__'] = (is_object($object)?$object->idprof1:'');
+				$substitutionarray['__THIRDPARTY_IDPROF2__'] = (is_object($object)?$object->idprof2:'');
+				$substitutionarray['__THIRDPARTY_IDPROF3__'] = (is_object($object)?$object->idprof3:'');
+				$substitutionarray['__THIRDPARTY_IDPROF4__'] = (is_object($object)?$object->idprof4:'');
+				$substitutionarray['__THIRDPARTY_IDPROF5__'] = (is_object($object)?$object->idprof5:'');
+				$substitutionarray['__THIRDPARTY_IDPROF6__'] = (is_object($object)?$object->idprof6:'');
+				$substitutionarray['__THIRDPARTY_TVAINTRA__'] = (is_object($object)?$object->tva_intra:'');
+				$substitutionarray['__THIRDPARTY_SKYPE__'] = (is_object($object)?$object->skype:'');
```
Comment (frederic34) [inline htdocs/core/lib/functions.lib.php]: @bb2a there is a dictionary for socialnetworks

Comment (bb2a) [reply htdocs/core/lib/functions.lib.php]: @frederic34 where is the dictionary

Comment (frederic34) [reply htdocs/core/lib/functions.lib.php]: @bb2a https://github.com/Dolibarr/dolibarr/pull/11867

Comment (bb2a) [reply htdocs/core/lib/functions.lib.php]: delete lines socialnetworks.

**Comment by atm-maxime** [comment]
When you open a PR, please write what it does or what is concerned about. Here you left the PR template text...

**Pull request merged**